module/testing.py

```diff
@@ -31,11 +31,14 @@ def lock_path(display):
 def find_display():
     display = 10
     while True:
-        f = open(lock_path(display), "w+")
         try:
-            fcntl.flock(f.fileno(), fcntl.LOCK_EX | fcntl.LOCK_NB)
+            f = open(lock_path(display), "w+")
+            try:
+                fcntl.flock(f.fileno(), fcntl.LOCK_EX | fcntl.LOCK_NB)
+            except:
```

Comment:
Can we still only catch OSError here, since that's all we expect?

Reply:
It doesn't really matter here, the only purpose of this `except` is to ensure that the file is closed if it's not going to be used.

Reply:
I'd still prefer to catch only what we expect.

Reply:
Hmm, I see lint doesn't like it for whatever reason.

Reply:
Changed to `OSError` now.

Reply:
Yeah, I'd just put the close in a `finally` block, no need to even catch anything at all really.

Reply:
If you used `finally`, you'd be returning a closed file descriptor on success.

Reply:
Oh, yep, derp :)